Tests/Functional/Service/Content/ContentMoveServiceTest.php

```
@@ -69,24 +71,130 @@ protected function tearDown(): void
+    {
+        $result = $this->subject->move(1, 201, null, 11);
+
+        self::assertFalse($result['success']);
+        self::assertSame(422, $result['statusCode']);
+        self::assertSame(0, (int) $this->readRecord(1)['colPos']);
+    }
+
+    #[Test]
+    public function moveRejectsContainerColumnWithoutContainerUid(): void
+    {
+        $before = $this->readRecord(1);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$before'. (undefined)

(UnusedLocalVariable)